node/src/components/consensus/era_supervisor.rs

```diff
@@ -256,6 +265,7 @@ where
         (self
             .protocol_config
             .last_activation_point
+            .saturating_add(1)
```

Comment:
Please add a comment (here and below, in `iter_past_other`) why we're adding 1 era to this. No one will remember that in a week.

Reply:
Added a clarifying comment in f377bdc7a